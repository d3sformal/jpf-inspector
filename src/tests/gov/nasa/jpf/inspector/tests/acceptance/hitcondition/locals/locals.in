cr bp local_access = outside
cr bp local_read = outside
rw
del bp 1 && del bp 2
cr bp local_write = outside
rw
del bp 3
cr bp class = *Callstack and method = middle and local_access = x
rw
del bp 4
cr bp it=lr or it=lw
rw
del bp 5
cr bp it=la
rw
del bp 6
rw

