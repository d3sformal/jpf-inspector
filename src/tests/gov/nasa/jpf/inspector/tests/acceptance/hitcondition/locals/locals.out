INFO: JPF created and connected, SuT is started
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=1 local_access=outside]
Breakpoint with ID 1 successfully deleted.
Breakpoint with ID 2 successfully deleted.
INFO: Breakpoint hit: [3 :  state=ENABLED hits=1 hitsTotal=1 local_write=outside]
Breakpoint with ID 3 successfully deleted.
INFO: Breakpoint hit: [4 :  state=ENABLED hits=1 hitsTotal=1 ( class = *Callstack ) and ( ( method = middle ) and ( local_access=x ) )]
	In class "gov.nasa.jpf.inspector.tests.acceptance.sut.Callstack".
	In method "middle".
Breakpoint with ID 4 successfully deleted.
INFO: Breakpoint hit: [5 :  state=ENABLED hits=1 hitsTotal=1 ( instruction_type=local_read ) or ( instruction_type=local_write )]
	SuT (Thread=0) executes the gov/nasa/jpf/inspector/tests/acceptance/sut/Callstack.java:23 - istore_2
Breakpoint with ID 5 successfully deleted.
INFO: Breakpoint hit: [6 :  state=ENABLED hits=1 hitsTotal=1 instruction_type=local_access]
	SuT (Thread=0) executes the gov/nasa/jpf/inspector/tests/acceptance/sut/Callstack.java:32 - istore_0
Breakpoint with ID 6 successfully deleted.
INFO: JPF Terminating